docs/getting-started/quick-start-guides/deploy-rancher-manager/aws-marketplace.md

```diff
@@ -9,6 +9,6 @@ description: Use Amazon EKS to deploy Rancher server.
 
 import YouTube from '@site/src/components/YouTube'
 
-Amazon Elastic Kubernetes Service (EKS) can quickly [deploy Rancher to Amazon Web Services (AWS)](https://documentation.suse.com/trd/kubernetes/single-html/gs_rancher_aws-marketplace/). To learn more, see our [Amazon Marketplace listing](https://aws.amazon.com/marketplace/pp/prodview-go7ent7goo5ae). Watch the demo for a walkthrough of AWS Marketplace SUSE Rancher setup:
+You can quickly deploy SUSE Rancher on Amazon Elastic Kubernetes Service (EKS.) To learn more, see the Usage Instructions under Usage Information in the [Amazon Marketplace listing](https://aws.amazon.com/marketplace/pp/prodview-f2bvszurj2p2c). 
```

Comment:
Just to confirm -- will Rancher Prime running in AWS be the only option listed in the Marketplace going forward?

Reply:
yes. Rancher Setup has been removed.